devices/elko/thermostat_16_a.json

```diff
@@ -0,0 +1,356 @@
+          "default": 0
+        },
+        {
+          "name": "config/mode"
+        },
+        {
+          "name": "config/offset"
+        },
+        {
+          "name": "config/on"
+        },
+        {
+          "name": "config/reachable"
+        },
+        {
+          "name": "state/lastupdated"
+        },
+        {
+          "name": "state/on",
+          "parse": {
+            "at": "0x0008",
+            "cl": "0x0201",
+            "ep": 1,
+            "eval": "Item.val = Attr.val > 3;",
+            "fn": "zcl"
+          },
+          "read": {
+            "fn": "none"
+          }
+        },
+        {
+          "name": "config/locked",
+          "refresh.interval": 3660
+        },
+        {
+          "name": "state/temperature",
+          "default": 0
+        }
+      ]
+    },
+      {
+         "type":"$TYPE_TEMPERATURE_SENSOR",
+         "restapi":"/sensors",
+         "uuid":[
+            "$address.ext",
+            "0x02",
+            "0x0402"
+         ],
+         "items":[
+            {
+               "name":"attr/id"
+            },
+            {
+               "name":"attr/lastannounced"
+            },
+            {
+               "name":"attr/lastseen"
+            },
+            {
+               "name":"attr/manufacturername"
+            },
+            {
+               "name":"attr/modelid"
+            },
+            {
+               "name":"attr/name"
+            },
+            {
+               "name":"attr/swversion"
+            },
+            {
+               "name":"attr/type"
+            },
+            {
+               "name":"attr/uniqueid"
+            },
+            {
+               "name":"config/offset",
+               "default":0
+            },
+            {
+               "name":"config/on"
+            },
+            {
+               "name":"config/reachable"
+            },
+            {
+               "name":"state/lastupdated"
+            },
+            {
+               "name":"state/temperature"
+            }
+         ]
+      },
+      {
+         "type":"$TYPE_TEMPERATURE_SENSOR",
+         "restapi":"/sensors",
+         "uuid":[
+            "$address.ext",
+            "0x03",
+            "0x0402"
+         ],
+         "items":[
+            {
+               "name":"attr/id"
+            },
+            {
+               "name":"attr/lastannounced"
+            },
+            {
+               "name":"attr/lastseen"
+            },
+            {
+               "name":"attr/manufacturername"
+            },
+            {
+               "name":"attr/modelid"
+            },
+            {
+               "name":"attr/name"
+            },
+            {
+               "name":"attr/swversion"
+            },
+            {
+               "name":"attr/type"
+            },
+            {
+               "name":"attr/uniqueid"
+            },
+            {
+               "name":"config/offset",
+               "default":0
```

Comment:
Could you please remove `"default": 0` and do not forget to also remove the comma in the line above :)